cl/search/models.py

```diff
@@ -2731,6 +2731,11 @@ class OpinionCluster(AbstractDateTimeModel):
         storage=IncrementingAWSMediaStorage(),
         blank=True,
     )
+    harvard_id = models.CharField(
+        help_text="The ID of the item in the Caselaw Access Project (Harvard)",
+        default=0,
```

Comment:
I'd expect null if we don't have the ID?

Reply:
🤔  Well!, this explains oddity of needing to use a string in my test – it should be a `IntegerField`!